charts/ocis/values.yaml

```diff
@@ -85,20 +85,22 @@ insecure:
   ocisHttpApiInsecure: false
 
 cache:
-  # -- Type of the cache to use. It defaults to no cache or a in memory cache depending on the service.
+  # -- Type of the cache to use. It defaults to no cache.
   # To disable the cache, set to "noop".
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
-  type: ""
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  type: "nats-js-kv"
   # -- Nodes of the cache to use.
-  nodes: []
-  # nodes:
-  #   - redis-master-1.ocis-redis.svc.cluster.local:6379
-  #   - redis-master-2.ocis-redis.svc.cluster.local:6379
+  nodes:
+    - nats.ocis-nats.svc.cluster.local:4222
+  # -- Username to use for authentication
+  authusername: ""
+  # -- Password to use for autentication
+  authpassword: ""
 
 store:
   # -- Configure the store type. To disable the cache, set to "noop".
   # Defaults to "nats-js". Therefore `store.nodes` defaults to the address of the builtin NATS.
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
   type: nats-js
```

Comment:
I explicitly didn't change this value because it needs data migration. Or am I missing something?

Reply:
I don't think that there is currently a way to migrate? At least I don't know any documentation about our cache / store implementations that document how to migration from one to another.

I'd honestly prefer to just break this. We're on a major 0 release for the Chart if we do one... So we're fine to break everything at every time